source/fuzz/transformation_split_block.cpp

```diff
@@ -135,7 +135,7 @@ void TransformationSplitBlock::Apply(
   // predecessor operand so that the block they used to be inside is now the
   // predecessor.
   new_bb->ForEachPhiInst([block_to_split](opt::Instruction* phi_inst) {
-    // The following assertion is a sanity check.  It is guaranteed to hold
+    // The following assertion is a coherence check.  It is guaranteed to hold
```

Comment:
"The following assertion is a coherence check."  Aren't all assert a "coherence check",  this sentence is not useful.  Something like: "|IsApplicable| should guarantee that the block has exactly one predecessor."  Even that might not be needed because of the message in the assert.

Reply:
Removed comment, and changed string associated with assert to make it clear that this is a precondition (that IsApplicable has established).